pkg/maps/lbmap/lbmap.go

```diff
@@ -629,6 +630,10 @@ func DumpServiceMapsToUserspace(includeMasterBackend, skipIPv4 bool) (loadbalanc
 			return
 		}
 
+		if k := svcValue.RevNatKey().GetKey(); k != uint16(0) {
```

Comment:
Please add comments indicating what is being done here / other parts below with idcache with the idcache, and why we would need to ignore a key with ID zero.